### DISCUSSION TITLE
Request for Features like Socket io ~ null safe >2 and Mongodb and webRTC support

### DISCUSSION THREAD

**Discussion by Manishmg3994** [serverpod/serverpod#354] [Q&A] [answered]
as i found your Ralic system Support web socket but , i wish you to add a support for socket.io client and  socket.io  server in that as websockets are good but limited as Proxy servers <VPN>  {websocket is different from socket io as socket io is advanced}

**Comment by vlidholt** [comment, marked as answer]
I moved this to a discussion as it is more of a question and not too specific. Feel free to open separate tickets for the different features that you are requesting.

Serverpod supports real-time communication and can be used as a messaging server for WebRTC. Socket.io is built on top of WebSocket, just like Serverpod. You can find more information here:
https://docs.serverpod.dev/concepts/streams

**Comment by Manishmg3994** [comment]
Thanks For Your Support , I will Try to Learn Serverpod and Solve those Complications , Stay with me 
# Thanks in Advance
